vendors/twtg/models/neon-vb-usc/model.yaml

```diff
@@ -0,0 +1,58 @@
+# Commercial name of the model 
+name: TWTG NEON Vibration Sensor (USC)
+# Functional description of the product. Maximum 500 characters. 
+description: "The NEON Vibration Sensor provides insight on vibrations (acceleration and velocity) of industrial equipment (using United States customary units). Triggers can be set, based on which the sensors can send alerts."
+# Logo of the device
+logo: twtg-vb.png
+# Product name & Version
+product:
+    name: TWTG/VB
+    version: 1
+# ID(s) of the profile that defines the LoRaWAN characteristics of this model.
+deviceProfileId: 
+    - twtg_us915_RP2-1.0.4_classA
+ 
+# You may optionally customize any of the following settings to override the generic value set in LoRaWAN device profiles associated with your model. Leave empty if you want to keep the Device Profile settings. 
+# Maximum device TX Conducted output power in dBm. 
+maxTxPower: 14
+# Minimum device TX Conducted output power in dBm. 
+minTxPower: 5
+# Maximum device TX Radiated output power in dBm.  
+maxTxEIRP: 6
+# Minimum device TX Radiated output power in dBm.  
+minTxEIRP: 4
+# Typical mobility profile of the device. Possible values are 'near_static' (also valid for static devices), 'walking_speed', 'vehicular_speed'  
+# or 'random' (not known, changes over time). 
+motionIndicator: NEAR_STATIC
+ 
+# Is your device certified by the LoRa Alliance? Possible values: true, false. 
+LoRaWANCertified: true
+# Always mandatory: `<vendorId>:<modelName>:<modelVersion>` (You should have only one per model)
+modelId: twtg:neon-vb-usc:1
+# <vendorId>:<protocolName>:<protocolVersion> Needed for linking the model with a specific driver -> must be the same one used in driver.yaml in the corresponding model (You might have several ones)
+protocolId: twtg:neon-vb-usc:3
+# DataSheet URL (optional) 
+specificationURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/blob/main/VB/6016_P20-002_Data-Sheet-NEON-Vibration-Sensor_C.pdf
+# User Guide URL (optional) 
+userGuideURL: https://github.com/TWTG-R-D-B-V/neon-product-documentation/tree/main/VB/Installation%20%26%20Use
+# Available sensors following Actility ontology: https://github.com/actility/thingpark-iot-flow-js-driver/blob/master/UNITS.md 
+sensors: 
+    - temperature:Far
+    - velocity:in/s
```

Comment:
`in` and `ft` were not listed, so I don't know if this is allowed. If not, I can just remove them.

Reply:
Hello @martinichka ,

At the moment, we can keep sensors compatibles with the existing ontology, so please remove the ones which are not listed. In the future, we can enhance this part by adding first these units to the ontology.

Reply:
I removed the non-existing unit